lib/srv/regular/sshserver_test.go

```diff
@@ -1837,3 +1867,20 @@ func waitForSites(s reversetunnel.Tunnel, count int) error {
 		}
 	}
 }
+
+// getMaxPipeSize returns the maximum size of the pipe buffer.
+func getMaxPipeSize() (int, error) {
```

Comment:
Does this work on macOS and anything else we support (somewhat)?

Reply:
Ah, good point, let me check.

Reply:
@xacrimon Updated. Hard coded it to be `65536 + 1` since it appears this is pretty difficult (if even possible) to get on macOS.